components/suggest/src/rs.rs

```diff
@@ -168,8 +168,24 @@ pub(crate) struct DownloadedAmpSuggestion {
     pub impression_url: String,
     #[serde(rename = "icon")]
     pub icon_id: String,
+    pub score: f64,
```

Comment:
According to the [schema](https://github.com/mozilla-services/quicksuggest-rs/blob/main/schema/quicksuggest.1.schema.json#L70-L75), `score` is optional meaning that it's possible to see `null` in the RS attachments. So we should make it `Option<f64>`. This also requires us to handle the `null` case for `PartialEq` and we can't implement `Eq` for it for total ordering.

Reply:
Feel free to remove the `PartialEq` and `Eq` derives from `DownloadedAmpSuggestion`, `DownloadedAmpWikipediaSuggestion`, and the other downloaded suggestion types, too!

They're here "just in case" (and will be optimized out as dead code), but we don't expose these types outside our crate, and we don't use them in our code or in unit tests, so we can just drop them 😊